project/example/example/main.cpp

```diff
@@ -90,6 +94,147 @@ int main()
 	BaseScene* scene = new BaseScene();
 	scene->Init();
 
+	FMOD::System     *system;
+	FMOD::Sound      *sound1, *sound2, *sound3;
+	FMOD::Channel    *channel = 0;
+	FMOD_RESULT       result;
+	unsigned int      version;
+	void             *extradriverdata = 0;
```

Comment:
this whole section in main probably shouldn't be here (main.cpp is for the core engine logic, ie. creating a window and calling the game to update), `BaseScene.cpp` would likely be a better place to put this